scripts/travis-ci.sh

```diff
@@ -39,7 +39,7 @@ if [ "$TRAVIS_OS_NAME" == "osx" ]; then
     type yarn > /dev/null || brew install yarn
     brew install nvm
     source /usr/local/opt/nvm/nvm.sh
-    nvm install 10.16.3 # install this node version
+    nvm install 14.15.4 # install this node version
```

Comment:
:+1: but now we need to push 10s of gigabytes of a new docker image... :sweat:

Reply:
@thisconnect before I start draining my harddrive, just want to be sure: no issues with node 14, right?

Reply:
actually, I can see macos CI build works now. let me update this in a separate PR because I also wanted to bump go version.

Reply:
> 👍 but now we need to push 10s of gigabytes of a new docker image... 😓

doesn't this happen once and then it is cached?
Else I don't quite get it. 14.15.4 needs 10s of gigabytes BUT 10.16.3 is fine?

Reply:
> actually, I can see macos CI build works now.

yep node 14 LTS 👍

Reply:
> no issues with node 14, right?

you are kidding wright?

Reply:
> doesn't this happen once and then it is cached?
> Else I don't quite get it. 14.15.4 needs 10s of gigabytes BUT 10.16.3 is fine?

The docker image is pulled every time CI runs. 10.16.3 is approx. the same size, so no difference really, tag version 6 at https://hub.docker.com/r/shiftcrypto/bitbox-wallet-app/tags?page=1&ordering=last_updated. I need to now build version 7 with the new node and push it to docker hub.